### DISCUSSION TITLE
Bar chart vertical axis issue

### DISCUSSION THREAD

**Discussion by dgwyer** [observablehq/plot#1684] [Q&A] [answered]
Just playing around with a simple bar chart and y-axis. I seem to be having an issue setting and styling the y-axis properly. The only way I found works is to set the `y: {line: true}` property but I also need to style this.

Here's an example notebook with more details of what I mean.

https://observablehq.com/d/78d8300093d577fd

**Comment by Fil** [comment, marked as answer]
try:

    Plot.frame({anchor: "left", stroke:"blue"})

https://observablehq.com/plot/marks/frame

**Reply by dgwyer** [reply]
Nice, thanks!
